query/view/proximity_boost.js

```diff
@@ -0,0 +1,7 @@
+const peliasQuery = require('pelias-query');
+const ngrams_strict = require('./ngrams_strict');
+
+module.exports = function (vs) {
+    const fuzziness = vs.var('fuzzy:fuzziness').get();
```

Comment:
Yes, integrating the influence of the "exact match" in a separate `should` clause is a valid alternative that I also though about and experimented with. The challenge here is finding the perfect `boost` for this - as it is a very thin margin between over-qualifying the exact matches and not actually changing anything.
In my experiments I saw that irrelevant / chaotic results come from this distance clause and I think the safe approach to it would be to nuance it with the "exact match" filter.
Also, historically the `focus` part of the `should` had a `ngrms_strict` filter for this (example from the original PR that proposed fuzzy, although this is not something introduced by the changes for fuzzy: https://github.com/pelias/api/blob/a2c5966ed7fdcbdca3cc527bf1dad0bb5a2cc5f1/query/autocomplete.js).